var/lib/delphix-platform/ansible/apply

```diff
@@ -36,13 +36,12 @@ function find_playbooks() {
 	# old directory. Once all consumers can be updated to install
 	# into the new directory, we can remove this logic.
 	#
-	if ! find "/etc/delphix-platform/ansible" \
+	find "/etc/delphix-platform/ansible" \
 		-maxdepth 2 \
 		-mindepth 2 \
-		-name playbook.yml | sort -n; then
-		echo "Failure when finding playbooks." 2>&1
-		exit 1
-	fi
+		-name playbook.yml | sort -n
+
+	return 0
```

Comment:
is this `return` call necessary?

Reply:
No, it isn't really necessary. I just wanted to point out that we were deliberately ignoring the exit status of `find`. Maybe this just makes it more confusing though. I can remove it.